UI/Theme.cpp

```diff
@@ -0,0 +1,238 @@
+// Copyright (c) 2013- PPSSPP Project.
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, version 2.0 or later versions.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License 2.0 for more details.
+
+// A copy of the GPL 2.0 should have been included with the program.
+// If not, see http://www.gnu.org/licenses/
+
+// Official git repository and contact information can be found at
+// https://github.com/hrydgard/ppsspp and http://www.ppsspp.org/.
+
+#include "ppsspp_config.h"
+
+#include <algorithm>
+
+#include "UI/Theme.h"
+
+#include "Common/File/FileUtil.h"
+#include "Common/File/VFS/VFS.h"
+#include "Common/Data/Format/IniFile.h"
+
+#include "Core/Config.h"
+
+#include "Common/UI/View.h"
+#include "Common/UI/Context.h"
+#include "Core/System.h"
+
+struct themeInfo {
```

Comment:
nit: Prefer ThemeInfo to stay consistent with other struct/class names.

-[Unknown]